code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -622,27 +622,32 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	if(active_uplink_check(user))
 		return
 
-	var/dat = {"<html><head><title>Personal Data Assistant</title></head><body bgcolor=\"#808000\"><style>a, a:link, a:visited, a:active, a:hover { color: #000000; }img {border-style:none;}</style>
-<a href='byond://?src=\ref[src];choice=Close'><img src=pda_exit.png> Close</a>"}
+	if(user.client)
+		var/datum/asset/simple/C = new/datum/asset/simple/pda()
+		send_asset_list(user.client, C.assets)
+
+	var/dat = list()
+	dat += {"
+	<html>
+	<head><title>Personal Data Assistant</title></head>
+	<body>
+	<link rel="stylesheet" type="text/css" href="pda.css"/> <!--This stylesheet contains all the PDA icons in base 64!-->
```

Comment:
Yes, this is rather primitive sadly. I know HTML datums handle this stuff by themselves, but I kept running into the same inscrutable problems with stylesheets hating my soul.